contracts/implementations/MultipleMinter.sol

```diff
@@ -46,8 +49,6 @@ contract MultipleMinter is
     function mintVehicleAndSdSign(
         MintVehicleAndSdInput calldata data
     ) external onlyRole(MINT_VEHICLE_SD_ROLE) {
-        NodesStorage.Storage storage ns = NodesStorage.getStorage();
```

Comment:
Cutting down on variables. I see.

Reply:
I wonder how long I'll be able to do this